components/webstories/Stories.tsx

```diff
@@ -68,13 +65,24 @@ const Stories = ({
     }
   }, [Story]);
 
+  useEffect(() => {
+    if (typeof window !== "undefined") {
+      const handleResize = () => setWindowWidth(window.innerWidth);
+      setWindowWidth(window.innerWidth);
+      window.addEventListener("resize", handleResize);
+      return () => window.removeEventListener("resize", handleResize);
+    }
+  }, []);
+
   const handlePauseResume = () => {
     paused(!isPaused);
     setIsPaused(!isPaused);
   };
 
   const handleSwipeUp = () => {
-    window.location.href = "/webstories";
+    if (typeof window !== "undefined") {
```

Comment:
can use next router instead of window.location

Reply:
Done